src/containers/AdminDashboard/AdminController.tsx

```diff
@@ -23,18 +25,32 @@ class AdminController {
     this.editPicAPI = this.editPicAPI.bind(this);
     this.editor = this.editor.bind(this);
     this.handleEditorChange = this.handleEditorChange.bind(this);
-    this.addForumButton = this.addForumButton.bind(this);
     this.createBook = this.createBook.bind(this);
     this.addAdminUser = this.addAdminUser.bind(this);
     this.onChangeYouthContent = this.onChangeYouthContent.bind(this);
     this.putAPI = this.putAPI.bind(this);
+    this.warnNotif = this.warnNotif.bind(this);
+    this.addForumButton = this.addForumButton.bind(this);
+  }
+  
+  warnNotif(id: string): void{
+    store.addNotification({
+      title: id,
+      message: 'Error, cannot dispatch',
```

Comment:
`message` needs to be a function param, Error, cannot dispatch is not helpful and needs to be specific